lauraguerra1/whats-cookin
# Feature/add ingredients info to panel

##Description
This pr adds ingredient data to the right side panel of the recipe.

<img width="405" alt="Screenshot 2023-05-18 at 5 17 01 PM" src="https://github.com/lauraguerra1/whats-cookin/assets/32660020/6796cb52-5dad-4dcb-b1a4-a4a3267aa1b9">

##Testing Notes
- Click on a recipe, you should see the corresponding amount and units of each ingredient required
  - If ingredients are long, they should word-wrap onto the next line
  - If ingredients list is super long, it will clip off the bottom of the page.  I don't have any scrolling implemented.
- check boxes function as expected
-  When recipe panel is closed, all ingredients are removed from the element and ready to accept the next set
